src/content/docs/apis/nerdgraph/examples/nerdgraph-api-secret-management-service.mdx

```diff
@@ -0,0 +1,255 @@
+---
+title: "NerdGraph tutorial: Secret Management Service"
```

Comment:
Global comment: Check out capitalization standard for Secret Management Service. I vaguely remember we use sentence case for feature names too. So, only Secret should be initial caps. Please double-check.

Reply:
- Pridhvi had spoken about a fifth mutation that we could use to restore the specific version of a deleted/purged secret. Is that out of scope now?
- Pridhbi had mentioned three NR solutions/capabilities for which this preview is supported: Incident management, Workflows, and I-Can't-Remember-Anymore. We need to have a small section/callout in each of those docs mentioning the secret mgmt srvc and link to this GraphQL API doc from there and also cross-link to each of those solution docs from this new topic.

Reply:
As discussed with Pridhvi, we don't limit our support for the mentioned capabilities so there is no need for a callout.

Reply:
> Global comment: Check out capitalization standard for Secret Management Service. I vaguely remember we use sentence case for feature names too. So, only Secret should be initial caps. Please double-check.

Thanks for pointing this out.